common/djangoapps/entitlements/api/v1/views.py

```diff
@@ -0,0 +1,53 @@
+import logging
+
+from django.utils import timezone
+from django_filters.rest_framework import DjangoFilterBackend
+from edx_rest_framework_extensions.authentication import JwtAuthentication
+from rest_framework import permissions, viewsets
+from rest_framework.authentication import SessionAuthentication
+
+from entitlements.api.v1.filters import CourseEntitlementFilter
+from entitlements.models import CourseEntitlement
+from entitlements.api.v1.serializers import CourseEntitlementSerializer
+from student.models import CourseEnrollment
+
+log = logging.getLogger(__name__)
+
+
+class EntitlementViewSet(viewsets.ModelViewSet):
+    authentication_classes = (JwtAuthentication, SessionAuthentication,)
+    permission_classes = (permissions.IsAuthenticated, permissions.IsAdminUser,)
+    queryset = CourseEntitlement.objects.all().select_related('user')
+    lookup_value_regex = '[0-9a-f-]+'
+    lookup_field = 'uuid'
+    serializer_class = CourseEntitlementSerializer
+    filter_backends = (DjangoFilterBackend,)
+    filter_class = CourseEntitlementFilter
+
+    def perform_destroy(self, instance):
+        """
+        This method is an override and is called by the DELETE method
+        """
+        save_model = False
+        if instance.expired_at is None:
+            instance.expired_at = timezone.now()
```

Comment:
will this save UTC?

Reply:
This will save as local time depending on the setting `settings.USE_TZ:`, which is currently True in the LMS.  This was a requested change from @clintonb I will check in with him and then update.

Reply:
Original comment from @clintonb 

> Make sure this value is timezone-aware. You may want to use timezone.now(). See https://docs.djangoproject.com/en/1.11/topics/i18n/timezones/.

Reply:
@bderusha and @clintonb 

I did a local test and the time stored in the DB is in UTC. I am going to squash and merge.

Reply:
I don't think that this database value should ever have the chance to be timezone aware.